snapcraft/cli/parts.py

```diff
@@ -25,9 +28,13 @@ def partscli(ctx):
 
 @partscli.command()
 @click.pass_context
-def update(ctx):
+def update(ctx, **kwargs):
     """Updates the parts listing from the cloud."""
-    remote_parts.update()
+    if env.is_containerbuild():
+        project_options = get_project_options(**kwargs)
+        lifecycle.containerbuild('update', project_options)
+    else:
+        remote_parts.update()
```

Comment:
This seems like a separate feature/bugfix, no?

Reply:
Same forum topic, arguably separate fix. I dropped it from this one and proposed #1546.